framework/source/class/qx/util/ResourceManager.js

```diff
@@ -232,7 +232,7 @@ qx.Class.define("qx.util.ResourceManager",
     toDataUri : function (resid)
     {
       var resentry = this.constructor.__registry[resid];
-      var combined = this.constructor.__registry[resentry[4]];
+      var combined = resentry?this.constructor.__registry[resentry[4]]:null;
```

Comment:
Could you add some spaces to make it more readable?

      var combined = resentry ? this.constructor.__registry[resentry[4]] : null;

Would be the way it seems to be used in 99% of all occurences in the framework ;-)